base_business_document_import/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016-2021 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Base Business Document Import",
+    "version": "15.0.1.0.0",
+    "category": "Tools",
+    "license": "AGPL-3",
+    "summary": "Provides technical tools to import sale orders or supplier invoices",
+    "author": "Akretion, Nicolas JEUDY, Odoo Community Association (OCA)",
+    "maintainers": ["alexis-via"],
+    "website": "https://github.com/OCA/edi",
+    "depends": [
+        # OCA/community-data-files
```

Comment:
Unused depend could be deleted.

Reply:
This is a comment about where (the repository) to find the below dependencies

Reply:
Okey, all good then.